### PR TITLE
feat(engine): generate new magic token for an existing configuration

### PR DESCRIPTION
**Pull request by bnjjj**
Signed-off-by: Benjamin Coenen <5719034+bnjjj@users.noreply.github.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/cmd_config.go [hunk 1]
```diff
@@ -314,3 +326,20 @@ var configSetCmd = &cobra.Command{
 		}
 	},
 }
+
+var configInitMagicTokenCmd = &cobra.Command{
+	Use:   "init-token",
+	Short: "Init magic token for a given CDS configuration file",
```
Comment (yesnault) [inline engine/cmd_config.go]:
```suggestion
	Short: "Generate a init-token for a given CDS configuration file",
```

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#11552.0 ✘  
* Build
  * Build and Package All ✘  
  * Engine Test with DB ✘  
  * tools/debpacker ✔ 
  * tools/os-ansible-inventory ✔ 
  * tools/smtpmock ✘  
  * UI Lint Translate ✔ 
  * UI Tests ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11552.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report cds-package#11552.0 ✘  
* build-package
  * Debian Engine Package ✘  
  * Docker Engine Package ✔ 
  * Package UI ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11558.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#11558.0 ■   
* Build
  * Build and Package All ✘  
  * Engine Test with DB ■   
  * tools/debpacker ✔ 
  * tools/os-ansible-inventory ✔ 
  * tools/smtpmock ✘  
  * UI Lint Translate ✔ 
  * UI Tests ✔

**Comment by ovh-cds** [comment]
CDS Report cds-package#11558.0 ✘  
* build-package
  * Debian Engine Package ✘  
  * Docker Engine Package ✔ 
  * Package UI ✔

**Pull request merged**